app/controllers/api/v1/users_controller.rb

```diff
@@ -17,6 +17,17 @@ def create
     end
   end
 
+  def register
+    user = User.new(user_params)
```

Comment:
New register action to save a user and make the session id equal to the user's id who just registered.